src/pages/Home.tsx

```diff
@@ -45,16 +48,162 @@ const HomePage: React.FC = () => {
         </IonToolbar>
       </IonHeader>
       <IonContent>
-
         <IonSlides pager={true} onIonSlideDidChange={ionSlideChanged}>
           <IonSlide>
-            01. Mike was here.
-    </IonSlide>
+            <IonCard>
+              <div className="concert bg"></div>
+              <IonCardHeader class="item item-header item-text-wrap">
```

Comment:
as you discovered, typically in React you need to use the className attribute to set classes on Html elements. We made the ionic components work with class as well to have compatibility with Preact, which does't have the className limitation.